machines/openId4VP/openId4VPServices.ts

```diff
@@ -0,0 +1,65 @@
+import {fetchKeyPair} from '../../shared/cryptoutil/cryptoUtil';
+import {__AppId} from '../../shared/GlobalVariables';
+import {constructProofJWT, OpenID4VP, OpenID4VP_Domain, OpenID4VP_Proof_Algo_Type} from '../../shared/openId4VP/OpenID4VP';
+
+export const openId4VPServices = () => {
+  const trustedVerifiersList = [
+    {
+      client_id: 'https://injiverify.dev2.mosip.net',
+      response_uri: [
+        'https://injiverify.qa-inji.mosip.net/redirect',
+        'https://injiverify.dev2.mosip.net/redirect',
+      ],
+    },
+    {
+      client_id: 'https://injiverify.dev1.mosip.net',
+      response_uri: [
+        'https://injiverify.qa-inji.mosip.net/redirect',
+        'https://injiverify.dev1.mosip.net/redirect',
+      ],
+    },
+  ];
```

Comment:
this won't work on new qa-inji env.
Let's expose an api in mimoto to get this trusted verifier list.

Reply:
same reason as below comment.